packages/terminal/src/browser/terminal-widget.ts

```diff
@@ -372,7 +404,7 @@ export class TerminalWidget extends BaseWidget implements StatefulWidget {
     }
 
     private async doResize() {
-        await Promise.all([this.waitForResized.promise, this.waitForTermOpened.promise]);
+        await Promise.all([this.waitForTermOpened.promise]);
```

Comment:
Is this change related to the issue addressed by this patch, or it's a separate issue?

In any case,

    await Promise.all([this.waitForTermOpened.promise]);

can probably be reduced to 

    await this.waitForTermOpened.promise;